theme/static/style/style.css

```diff
@@ -177,6 +177,12 @@ nav button {
     margin-bottom: 2rem;
 }
 
+/* make blog posts' hero images responsive */
```

Comment:
You can make this comment a bit more generic b/c it will apply to all images or remove it entirely.